src/classes/Advancement_Info.cls

```diff
@@ -70,35 +70,45 @@ public with sharing class Advancement_Info {
             hs.Advancement_Namespace__c = String.valueOf(Hierarchy_Settings__c.Advancement_Namespace__c.getDescribe().getDefaultValueFormula()).remove('\"');
         }
 
-        //Check if Advancement is installed
-        try {
-            Schema.DescribeSObjectResult[] descResult = Schema.describeSObjects(new String[]{hs.Advancement_Namespace__c + '__Advancement_Setting__mdt'});
-            isManaged = true;
-        } catch (Exception e) {
-            try {
-                Schema.DescribeSObjectResult[] descResult = Schema.describeSObjects(new String[]{'Advancement_Setting__mdt'});
-            } catch (Exception ex) {
-                setupCalled = true;
-                return;
+        //Check if GEM is installed
+        List<String> settingsObjs = new List<String> {'Advancement_Setting__mdt', 'GEM_API_Settings__mdt'};
+        List<String> useGEMFields = new List<String> {'Use_Advancement__c', 'Use_GEM__c'};
+
+        boolean found = false;
+
+        Integer i=0;
+        while (!found && i < settingsObjs.size()) {
+            try { // With Namespace
+                Schema.DescribeSObjectResult[] descResult = Schema.describeSObjects(new String[]{hs.Advancement_Namespace__c + '__' + settingsObjs[i]});
+                isManaged = true;
+                found = true;
+            } catch (Exception e) {
+                try { // Without Namespace
+                    Schema.DescribeSObjectResult[] descResult = Schema.describeSObjects(new String[]{settingsObjs[i]});
+                    found = true;
+                } catch (Exception ex) {
+                    i++;
+                }
             }
         }
 
+        if (!found) {
+            setupCalled = true;
+            return;
+        }
+
+       String gemAPISettingsObj = settingsObjs[i];
```

Comment:
Check your spacing here. Looks like the second line is in one space and then the rest follow suit.